packages/compartment-mapper/src/node-modules.js

```diff
@@ -383,6 +401,7 @@ const graphPackage = async (
  * @param {string} name - name of the package of interest.
  * @param {Set<string>} tags
  * @param {boolean} optional - whether the dependency is optional
+ * @param {Object} [commonDependencyDescriptors] - dependencies to be added to all packages
```

Comment:
Let’s narrow this type. I believe this is `Record<string, {specifier: string, alias: string}>`. `Object` and `object` are aliases for `any` in JSDoc.